frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -285,12 +285,6 @@ class ProgramStructure::FixupExtern : public Modifier {
 };
 
 void ProgramStructure::createExterns() {
-    for (auto r : registers) {
```

Comment:
Well, the back-end does not need to generate code for direct registers anywhere.
Where are they supported?